projects/packages/jetpack-mu-wpcom/src/features/wpcom-site-menu/wpcom-site-menu.php

```diff
@@ -544,3 +544,42 @@ function wpcom_maybe_enable_link_manager() {
 	update_option( 'link_manager_check', time() );
 }
 add_action( 'init', 'wpcom_maybe_enable_link_manager' );
+
+/**
+ * Add the Scheduled Updates menu item to the Plugins menu.
+ *
+ * Limited to sites with scheduled updates feature.
+ */
+function wpcom_add_scheduled_updates_menu() {
+	// Bail on Simple sites
+	if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
+		return;
+	}
+
+	/**
+	 * Don't show `Scheduled Updates` to administrators without a WordPress.com account being attached,
+	 * as they don't have access to any of the pages.
+	 */
+	if ( ! current_user_has_wpcom_account() ) {
+		return;
+	}
+
+	if (
+		function_exists( 'wpcom_site_has_feature' ) &&
```

Comment:
I think we'd want to return if the function doesn't exist

Reply:
You're right, added the checks, thanks!